src/db/utils/jobs/migration/__tests__/ConvertTickDateClimbedToDate.ts

```diff
@@ -0,0 +1,58 @@
+import { ApolloServer } from 'apollo-server'
+import muuid from 'uuid-mongodb'
+import TickDataSource, { createInstance as createTickInstance } from '../../../../../model/TickDataSource'
+import { setUpServer } from '../../../../../utils/testUtils.js'
+import { muuidToString } from '../../../../../utils/helpers.js'
+import { onConnected } from '../ConvertTickDateClimbedToDate'
+
+describe('ConvertTickDateClimbedToDate migration', () => {
+  let server: ApolloServer
+  let user: muuid.MUUID
+  let userUuid: string
+  let inMemoryDB
+
+  // Mongoose models for mocking pre-existing state.
+  let ticks: TickDataSource
+
+  beforeAll(async () => {
+    ({ server, inMemoryDB } = await setUpServer())
+    user = muuid.mode('relaxed').v4()
+    userUuid = muuidToString(user)
+
+    await inMemoryDB.clear()
+    ticks = createTickInstance()
+  })
+
+  afterAll(async () => {
+    await server.stop()
+    await inMemoryDB.close()
+  })
+
+  // This will throw an error due to multiple Mongoose connections.
```

Comment:
If you know a workaround here, please let me know! I'm not familiar with how these `connectDB` migrations work. Ideally we'd rewired it so we can run the migration functions decoupled from the Mongo `onConnect` event.